views/js/controller/tests/editTest.js

```diff
@@ -29,19 +29,26 @@ define([
     'use strict';
 
     return {
-
-
         /**
          * Controller's entrypoint
          */
         start(){
 
             const config = module.config();
-            const previewAction = actions.getBy('test-preview');
-            if (previewAction) {
-                previewAction.state.disabled = !config.isPreviewEnabled;
-                actions.updateState();
+
+            const getPreviewId = idx => `test-preview${idx ? `-${idx}` : ''}`;
+            const previewActions = []
+            for (let i = 0; i < 10; i++) {
```

Comment:
I see these logics are repeated in the item extension too. I've left some comments there that may apply here too (like, if the fixed `10` may become a problem later). 

Would it be possible to have this duplicated logic centralized in some place instead?

Reply:
Indeed, not the best. I've updated it with a constant instead.